jetty-core/jetty-rewrite/src/main/java/org/eclipse/jetty/rewrite/handler/Rule.java

```diff
@@ -35,7 +35,7 @@
      * @return the possibly wrapped {@code Request} and {@code Processor}, or {@code null} if the rule did not match
      * @throws IOException if applying the rule failed
      */
-    public abstract Processor matchAndApply(Processor input) throws IOException;
```

Comment:
Change line 35 to:
`     * @return the possibly wrapped {@code Request} and {@code Handler}, or {@code null} if the rule did not match`

Or maybe replace `{@code ...}` with `{@link ...}`?